modules/akmods/README.md

```diff
@@ -13,15 +13,10 @@ Ublue-os-akmods-addons & ublue-os-nvidia-addons are already included when necess
 
 To use the `akmods` module, specify the kernel modules you wish to install in the `install:` section of your recipe/configuration file.
 
-By default, the `akmods` module installs the `main` akmods for the `latest` version of Fedora.
+By default, the `akmods` module installs the `main` version of akmods.
 `main` akmods are also compatible with other images, except `surface(-nvidia)` & `asus(-nvidia)`.
 
-If you want to install akmods for `surface(-nvidia)` or `asus(-nvidia)` images, or for `older version of Fedora`, change this part in the Containerfile:
-
-```
-# Change this if you want different version/tag of akmods.
-COPY --from=ghcr.io/ublue-os/akmods:main-39 /rpms /tmp/rpms
-```
+If you want to install akmods for `surface(-nvidia)` or `asus(-nvidia)` images, change `base` entry in the recipe file.
```

Comment:
```suggestion
If you want to install akmods for `surface` or `asus` images, change `base` entry in the recipe file.
```

Removing nvidia here so we don't confuse `bluebuild` users with using `base` as this will try to pull an invalid image.